src/cephadm/cephadmlib/context.py

```diff
@@ -73,3 +74,14 @@ def __setattr__(self, name: str, value: Any) -> None:
             setattr(self._args, name, value)
         else:
             super().__setattr__(name, value)
+
+
+class DeploymentType(Enum):
```

Comment:
make a new file, maybe "deploy.py" that starts with just this

Reply:
done